lib/synchronizer.rb

```diff
@@ -44,6 +49,10 @@ def _sync(adapters)
           next
         end
       end
+      puts '--------------------'
+      puts "Update finished!"
+      puts "#{successfull.join(', ')}: sync success" unless successfull.empty?
+      puts "#{failed.join(', ')}: sync failed" unless failed.empty?
```

Comment:
`successful` and `failed ` arrays both may not be empty at the same time. And this condition will not work. I can write
```ruby
if !condition1
 lalala
end
if !condition2
 lalala
end
```
But I think it will look less beautiful than it is now.